dask_expr/tests/test_collection.py

```diff
@@ -799,8 +837,9 @@ def test_drop_duplicates(df, pdf):
     assert_eq(df.drop_duplicates(subset=["x"]), pdf.drop_duplicates(subset=["x"]))
     assert_eq(df.x.drop_duplicates(), pdf.x.drop_duplicates())
 
-    with pytest.raises(KeyError, match=re.escape("Index(['a'], dtype='object')")):
-        df.drop_duplicates(subset=["a"])
+    if BACKEND == "pandas":
+        with pytest.raises(KeyError, match=re.escape("Index(['a'], dtype='object')")):
```

Comment:
Why is this not failing for cudf?

Reply:
Does it seem okay to just catch the KeyError, and to check that `"'a'"` is in the message?

Reply:
Yeah that's fine by me